### DISCUSSION TITLE
Exploring WalletConnect 2.0: Seeking Vanilla JS Sample Code for Seamless Upgrades

### DISCUSSION THREAD

**Discussion by orangeqq** [WalletConnect/walletconnect-monorepo#2911] [Developer Support] [answered]
I encountered difficulties while upgrading from version 1.0 to 2.0 because the documentation did not provide any sample code in vanilla JS. Previously, I found this article very helpful: [link](https://github.com/WalletConnect/web3modal-vanilla-js-example). Does version 2.0 have similar sample code available?

**Comment by xzilja** [comment, marked as answer]
All examples are available here https://github.com/WalletConnect/web3modal-examples

**Reply by orangeqq** [reply]
I will check it. Thank you very much!
